src/client/pages/signup/signup.jsx

```diff
@@ -83,11 +101,13 @@ export default class Signup extends PureComponent {
       <Layout state={states.green}>
         <Layout.HeaderContent state={states.green}>
           <Auth className={styles.panel}>
-            {this.state.signupSuccess ? (
+            {signupSuccess ? (
               <div className={styles['signup-success']}>
                 <h2>Thank you for signing up!</h2>
                 <p>
-                  The verification email has been sent to {this.state.email}. <br />
+                  The verification email has been sent to
+                  { email }.
```

Comment:
It's better not to use spaces in curly braces when using string literals (and in other places of this PR)